lib/components/user/user-account-screen.js

```diff
@@ -127,27 +124,29 @@ class UserAccountScreen extends Component {
             // can be wired to be managed by Formik.
             (formikProps) => {
               const DisplayComponent = isCreating
-                ? NewAccountWizard
-                : ExistingAccountDisplay
+                ? lazy(() => import('./new-account-wizard'))
+                : lazy(() => import('./existing-account-display'))
```

Comment:
A deployment that has middleware persistence enabled will used either one, maybe it is `UserAccountScreen` that needs to be lazy-loaded from `ResponsiveWebApp`?

Reply:
I tried doing lazy loading in `ResponsiveWebApp` and ran into problems with `<suspense>` placement. I think best practice is to split the app up into as many files as possible anyway, especially with `HTTP/3 and QUIC`

Reply:
Are you/were you able to lazy load `UserAccountScreen` from `ResponsiveWebApp` based on config, without using `lazy`?

Reply:
Oh sorry I misread your comment -- React components must be code-split using `lazy`